core/commands/bitswap.go

```diff
@@ -6,17 +6,18 @@ import (
 	"io"
 
 	cmds "github.com/ipfs/go-ipfs/commands"
+	e "github.com/ipfs/go-ipfs/core/commands/e"
```

Comment:
I'm not offended by `e` , other things have weird single letter imports. I agree there really isnt a good name for it. both `err` and `errors` arent good, and `eror` feels wonky. Maybe `ce` for commands errors?